demos/apps/echo-with-auth/istio-manager.yaml

```diff
@@ -0,0 +1,51 @@
+# Manager service for discovery
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: istio
+data:
+  mesh: |-
+    authPolicy: 1                        # Enables Istio auth
+    mixerAddress: istio-mixer:9091
+    discoveryAddress: istio-manager:8080
+---
+apiVersion: v1
+kind: Service
+metadata:
```

Comment:
I would say deduplicate the entire file with the official manager install files. Easier way would be to add these lines to official istio-manager.yaml, but keep them commented, or have a istio-manager-with-auth.yaml file.